testing/export/export_package-build-with-components-and-version.cmake

```diff
@@ -0,0 +1,40 @@
+#=============================================================================
+# Copyright (c) 2018-2021, NVIDIA CORPORATION.
```

Comment:
The copyright years are off on all (or at least most) of the new files in testing/export.